docs/05-Development-and-CICD/01-duration-terms.adoc

```diff
@@ -4,7 +4,7 @@
 |===
 
 === Begriffe und Konzepte
-Development Environment, CI/CD Environment, Mean Time To Recovery (MTTR), Application Lifecycle Management, Formen von Deployments wie Rolling-, Canary- und Blue/Green Deployment, Cluster Design, Logging, Monitoring, Metriken, Distributed Tracing, Time Series Queries, Alerting
```

Comment:
In my opinion we should not move MTTR completely to chapter Automation and Operation, cause the base for it are streamlined images and tests.

Reply:
You are right, I did bring it back in commit [c39b6b9](https://github.com/isaqb-org/curriculum-cloudinfra/pull/27/commits/c39b6b94e1dbf8ffed2d51efbfceeab96394fdfb) :+1: